SmartcarAuth/OEM/OEM.swift

```diff
@@ -78,3 +78,17 @@ public class OEM {
     }
     
 }
+
+extension Array where Element == OEMName {
```

Comment:
Can we add a test for this?

Reply:
I think there is also a build error here

```
error: same-type requirement makes generic parameter 'Element' non-generic
extension Array where Element == OEMName {
```

Reply:
It builds on Xcode 9.1 and Xcode 8.3.3, both simulator, tests and device. This seems to be supported as of Swift 3.1 which is the oldest usable version of Swift, cf comment in https://stackoverflow.com/a/41663958/1439489